tests/OpenApiReaderTest.php

```diff
@@ -89,4 +89,10 @@ public function testGetOperationResponseHeaders()
         $this->assertNull($headers['x-not-defined']);
         $this->assertSame(['x-next', 'x-response-id'], array_keys($headers));
     }
+
+    public function testReadReferencedValues() {
+        $openapi = new OpenApiReader(__DIR__.'/ref.json');
+        $response   = $openapi->getOperationResponse('/api/v4/admin/loyalty/brands/{brand}/vip/rewards/analytics', 'get', 500);
+        $this->assertArrayHasKey('x-response-id', $response->getHeaders());
```

Comment:
This will fail. Although headers are case insensitive, the array returned by `$response->getHeaders()` uses the name defined in the openapi file as key. Which in this case is `X-Response-Id`